openshift/template.yaml

```diff
@@ -35,6 +37,8 @@ objects:
           env:
           - name: CAD_EXPERIMENTAL_ENABLED
             value: ${CAD_EXPERIMENTAL_ENABLED}
+          - name: LOG_LEVEL
```

Comment:
should we add the env to the cad container as well? I would like to see it configurable on both ;)
L320
```
kind: Task
  metadata:
    name: cad-checks
```